### ISSUE TITLE
TaurusForm all labels shown as "label"

### ISSUE THREAD

**Issue by jairomoldes** [taurus-org/taurus#696]
All labels in TaurusForm are shown as the string "label" if you add them to a GUI using "createPanel()". This doesn't happen if you use "setCentralWidget()".

The problem has been introduced in develop between 3cd8accc4359750014949d6d6dd2d0b4acdb1c6d (it worked fine) and 29410eae3626e8e8c6af876a7e766adbf559d8af (the problem is there).

You can easily reproduce the problem with this code:

```python
#!/usr/bin/env python

import sys
from taurus.core.util import argparse
from taurus.qt.qtgui.application import TaurusApplication
from taurus.qt.qtgui.panel import TaurusForm
from taurus.qt.qtgui.taurusgui import TaurusGui


def build_GUI():
    gui = TaurusGui()

    model = ['sys/tg_test/1/boolean_scalar', 'sys/tg_test/1/double_scalar']
    form1 = TaurusForm(gui, withButtons=False)
    form1.setModel(model)
    model = ['sys/tg_test/1/boolean_scalar', 'sys/tg_test/1/double_scalar']
    form2 = TaurusForm(gui, withButtons=False)
    form2.setModel(model)

    gui.createPanel(form1, 'Test form', permanent=True)
    gui.setCentralWidget(form2)
    gui.loadSettings()
    return gui

def main():
    #create parser, app, gui
    parser = argparse.get_taurus_parser()
    app = TaurusApplication(sys.argv, cmd_line_parser=parser,
                      app_name='test', app_version='1.0.0',
                      org_domain='ALBA', org_name='ALBA')

    #build and start GUI
    gui = build_GUI()
    gui.show()
    sys.exit(app.exec_())


if __name__ == '__main__':
    main()

**Comment by jairomoldes** [comment]
UPDATE!!!!

It looks like this is related with GUI settings stored in "ini" file.

**Comment by cpascual** [comment]
Hi, I cannot reproduce it with the given code in the latest develop (or with our internal package based on commit c80da9). I get the following result:

![image](https://user-images.githubusercontent.com/4115224/36471940-86daaeea-16f0-11e8-893f-afa9fb813980.png)

**Comment by cpascual** [comment]
Note, working with an internal GUI that originally caused this issue, we found that the problem could be solved by editing the ini file with taurusconfigbrowser and removing the TaurusValue configurations (`<perspective>/<taurusformpanel>widget/__item<X>`).

This indicates that the configuration has at some point stored some value that is not compatible with the current code. It would be nice to be able to reproduce it in order to fix it.

**Comment by cmft** [comment]
I could reproduce the error if a generate the .ini file with taurus 3 and after I launch the test with taurus 4

**Comment by cpascual** [comment]
> I could reproduce the error if a generate the .ini file with taurus 3 and after I launch the test with taurus4

Good. We can then see if there is some way to support the old ini files. 

But note that although we do try to preserve compatibility if possible, we do not offer any guaranty for ini files to be portable across versions (not even minor versions).
In other words: ini files should be considered volatile, and any critical changes done to a GUI should be made permanent (e.g. by exporting to a config.xml file)

**Comment by jairomoldes** [comment]
It is not necessary to use taurus3 to reproduce the problem. It can be reproduced with taurus 4:
- Checkout commit 3cd8accc4359750014949d6d6dd2d0b4acdb1c6d
- Open the sample GUI and close it
- Checkout commit 29410eae3626e8e8c6af876a7e766adbf559d8af
- Open the sample GUI: the problem is there.

**Comment by cpascual** [comment]
> It is not necessary to use taurus3 to reproduce the problem. It can be reproduced with taurus 4

I confirm that I was able to reproduce by creating the ini file with 3cd8acc and reading it with the soon-to-be new release (4.3.0) 387417fba
*But*, I just tested with the previous officially released version (4.1.1) and the problem does not seem to affect it (i.e. a file created with 4.1.1 is ok when read with 4.3.0).

Therefore IMHO this problem is relatively non-urgent since:

a) there is a way to repair the affected ini files
b) it requires a combination of a specific **unreleased** version with the coming version

**Comment by cpascual** [comment]
Still, I think that it is worth to keep this open till we debug the problem and learn to avoid similar cases in the future. For the moment, I upload here 2 .ini files: one created with 3cd8acc and another with 387417fba

Some further investigation comparing an affected ini file (created with 3cd8acc) and an unaffected one (created with 387417f) shows that the only difference is in the value stored in the `labelConfig` property of the TaurusValue. The affected version stores `"label"` and the unaffected one stores `"{attr.label}"`, as shown in this snapshot

![image](https://user-images.githubusercontent.com/4115224/36490133-de542336-1727-11e8-82e1-40cea7703d1d.png)
